# HG changeset patch
# User rk
# Date 1285925821 -7200
# Node ID bde2a2fa1c299365a268133d20a39e30589bdf28
# Parent  e2ae70bfa52d993daef46aee884d4d4aeaa34394
set the column member of the event object for wxEVT_COMMAND_DATAVIEW_ITEM_ACTIVATED and wxEVT_COMMAND_DATAVIEW_ITEM_CONTEXT_MENU

diff -r e2ae70bfa52d -r bde2a2fa1c29 src/generic/datavgen.cpp
--- a/src/generic/datavgen.cpp	Fri Oct 01 11:31:23 2010 +0200
+++ b/src/generic/datavgen.cpp	Fri Oct 01 11:37:01 2010 +0200
@@ -3622,6 +3622,8 @@ void wxDataViewMainWindow::OnMouse( wxMo
                 wxWindow *parent = GetParent();
                 wxDataViewEvent le(wxEVT_COMMAND_DATAVIEW_ITEM_ACTIVATED, parent->GetId());
                 le.SetItem( item );
+                le.SetColumn( col->GetModelColumn() );
+                le.SetDataViewColumn( col );
                 le.SetEventObject(parent);
                 le.SetModel(GetOwner()->GetModel());
 
@@ -3692,6 +3694,8 @@ void wxDataViewMainWindow::OnMouse( wxMo
         wxWindow *parent = GetParent();
         wxDataViewEvent le(wxEVT_COMMAND_DATAVIEW_ITEM_CONTEXT_MENU, parent->GetId());
         le.SetItem( item );
+        le.SetColumn( col->GetModelColumn() );
+        le.SetDataViewColumn( col );
         le.SetEventObject(parent);
         le.SetModel(GetOwner()->GetModel());
         le.SetValue(value);
